supervisor.go

```diff
@@ -119,9 +120,17 @@ func (s *SupervisorService) GetAuditHistoryAll() (map[string]SupervisorAuditHist
 
 // Suspend calls druid Supervisor service's Suspend API.
 // https://druid.apache.org/docs/latest/api-reference/supervisor-api/#suspend-a-running-supervisor
-func (s *SupervisorService) Suspend(string) (OutputIngestionSpec, error) {
-	var res OutputIngestionSpec
-	return res, errors.New("method Suspend not implemented")
+func (s *SupervisorService) Suspend(supervisorID string) (OutputIngestionSpec, error) {
+	r, err := s.client.NewRequest(http.MethodPost, applySupervisorId(supervisorSuspendEndpoint, supervisorID), nil)
+	var result OutputIngestionSpec
+	if err != nil {
+		return result, err
+	}
```

Comment:
Style/Optional:

1) Usually, error handling should come right after the call that returns error.
2) I'd say it's usually safer to return explicit zero value in err branches. 

(applies to `Resume` as well)

Reply:
fixed